tests/start_test.py

```diff
@@ -1,17 +1,29 @@
 import unittest
-
+from app.app_data.user_data_manager import TestUserDataManager
+from app.motivator.users.bot_users import NewBotUser, KnownBotUser
```

Comment:
`NewBotUser` and `KnownBotUser` can be imported from `app.motivator.users.user_builder`.